app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def update_post_counter_in_user
+    users.post_counter.update
+  end
```

Comment:
- [ ] [OPTIONAL] It would have been nice if you had the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` to call the `#update_post_counter_in_user` method every time a new post has been saved. You can read more on it.

Reply:
Okay, Thank you. I will implement it in my next project